### DISCUSSION TITLE
Android Studio Sync error

### DISCUSSION THREAD

**Discussion by FelixZhang00** [Tencent/tgfx#36] [Q&A] [answered]
error message: 
`Cannot invoke (class=AddRunTaskListener, method=onImportFinished, topic=project data import listener)`

![image](https://github.com/Tencent/tgfx/assets/5494156/28951a9a-b863-43ee-9227-e5d71bd8dae4)

Android Studio version :
```
Android Studio Iguana | 2023.2.1 Canary 8
Build #AI-232.9921.47.2321.10958940, built on October 17, 2023
Runtime version: 17.0.8+0-17.0.8b1000.22-10799086 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 13.4
```

**Comment by domchen** [comment, marked as answer]
I have just verified that the android demo works flawlessly on my computer. It appears that the issue you are encountering is related to your IDE, rather than the tgfx project.  Please verify if you have made any changes to the gradle configuration. Avoid clicking on the gradle upgrade suggestions. If you haven't made any modifications, you should be good. However, if you have, resolving the issue might involve deleting the Android directory and resetting it using git. If there are any other problems, they are most likely related to your IDE. Please conduct an online search to find relevant information and solutions.

**Reply by FelixZhang00** [reply]
What version of android studio are you using?

**Reply by domchen** [reply]
My Android Studio version is : Giraffe | 2022.3.1 Patch 3

**Comment by domchen** [comment]
I will transfer this Issue to Discussion, please ask any usage questions there in the feature. We only use Issues to track bugs.
